pep-0649.rst

```diff
@@ -713,25 +740,12 @@ makes some modifications to the annotations before it returns them.
 This PEP adds a new keyword-only parameter to these two functions,
 ``format``.  ``format`` specifies what format the values in the
 annotations dict should be returned in.
-``format`` accepts the following values, defined as attributes on the
-``inspect`` module::
-
-    VALUE = 1
-    FORWARDREF = 2
-    SOURCE = 3
+The ``format`` parameter on these two functions accept the same values
```

Comment:
```suggestion
The ``format`` parameters on these two functions accept the same values
```
(Or alternatively, add an s to "accept".)

Reply:
The 's' on accept is what I want, thanks.

Reply:
I concede it's a little weird that the sentence starts out talking about "The format parameter on these two functions", then after the semicolon talks about "these format parameters".  If you have a suggestion that makes it read better, by all means!

Reply:
I think the current wording is fine.